Sources/Logging/Logging.swift

```diff
@@ -721,6 +773,10 @@ public struct StreamLogHandler: LogHandler {
     }
 
     private func timestamp() -> String {
+        #if canImport(WASILibc)
+        // WASI doesn't have `strftime`, but Foundation `DateFormatter`
+        return sharedDateFormatter.string(from: Date())
```

Comment:
`swift-log` isn't meant to depend on `Foundation` so we'd need a different solution here.

Reply:
No idea, I thought it might be OK for this specific use case.

Reply:
Looks like they use `strftime` themselves for the replacement DateFormatter, so it seems they just don't re-export that. Which is a little weird: https://github.com/swiftwasm/swift-corelibs-foundation/blob/swiftwasm/Sources/Foundation/DateFormatter.swift#L538

Reply:
ha, that's weird. Let's ping @MaxDesiatov maybe he knows what's going on there?

Reply:
`strftime` does work, I don't see `import WASILibc` anywhere here though for these to be imported